assets/component-facets.css

```diff
@@ -744,20 +810,20 @@ details.menu-opening .mobile-facets__close svg {
 }
```

Comment:
This is happening on mobile: https://screenshot.click/12-04-wo88b-bk2tr.mp4

Reply:
Good call out. I had added some styles that are no longer necessary. Instead, I got rid of those styles and set the width to 100% (to match the grid cell) in the case that we're inside of the `.facets-layout-grid`. That way, if it's used by itself it should have the 9rem, otherwise it fills the full width.

Note: the reason this is necessary is our "default" should be 9rem (if this is used outside of filters), but that doesn't actually fill the width for our desktop filters. If we don't force it to 100% there, we will get extra spacing on the left/right, which we don't want, so I can just force it to 100% width.

<img width="357" alt="image" src="https://github.com/Shopify/dawn/assets/1019769/2301b2fc-fa95-4bee-b6fc-e6082941c4fb">